bigbluebutton-html5/imports/ui/components/cursor/service.js

```diff
@@ -35,13 +35,29 @@ export function publishCursorUpdate(payload) {
 }
 
 export function initCursorStreamListener() {
-  logger.debug({
+  logger.info({
     logCode: 'init_cursor_stream_listener',
+    extraInfo: { meetingId: Auth.meetingID, userId: Auth.userID },
   }, 'initCursorStreamListener called');
 
-  if (!cursorStreamListener) {
-    cursorStreamListener = new Meteor.Streamer(`cursor-${Auth.meetingID}`, { retransmit: false });
+  /**
+  * We create a promise to add the handlers after a ddp subscription stop.
+  * The problem was caused because we add handlers to stream before the onStop event happens,
+  * which set the handlers to undefined.
+  */
+  cursorStreamListener = new Meteor.Streamer(`cursor-${Auth.meetingID}`, { retransmit: false });
 
+  const startStreamHandlersPromise = new Promise((resolve) => {
+    const checkStreamHandlersInterval = setInterval(() => {
+      const streamHandlersSize = JSON.parse(JSON.stringify((Meteor.StreamerCentral.instances[`cursor-${Auth.meetingID}`].handlers)));
```

Comment:
This looks expensive. Is there no way to do something like 
`Meteor.StreamerCentral.instances[`cursor-${Auth.meetingID}`].handlers.length`?

Reply:
![Screenshot from 2020-01-13 09-44-23](https://user-images.githubusercontent.com/6312397/72264811-5486e480-35e9-11ea-9933-d37facb3fb97.png)

Reply:
Or actually 
![Screenshot from 2020-01-13 09-46-55](https://user-images.githubusercontent.com/6312397/72264973-ab8cb980-35e9-11ea-853d-9e83b3ad59d5.png)